### PR TITLE
Close RTCPeerConnections used for probing, and discard the freedom module.

### PR DESCRIPTION
**Pull request by bemasc**
[Review on Reviewable](https://reviewable.io/reviews/freedomjs/freedom-port-control/5)

### CODE REVIEW

**Comment by lally** [comment]
:+1:

File: src/utils.js [hunk 1]
```diff
@@ -81,6 +81,9 @@ var getPrivateIps = function () {
   // Gather candidates for 2 seconds before returning privateIps or timing out
   return new Promise(function (F, R) {
     setTimeout(function () {
+      pc.close().then(function () {
```
Comment (trevj) [inline src/utils.js]: Cool. I think to be sure to be sure you should call this on failure too, e.g.:

``` typescript
var cleanup = function() {
  freedom['core.rtcpeerconnection'].close(pc);
};
pc.close().then(cleanup, cleanup);
```

(my kingdom, etc., for `finally` blocks)

Comment (bemasc) [inline src/utils.js]: Done!

**Comment by trevj** [comment]
:+1:

**Pull request merged**